### PR TITLE
Added bootstrap4 theme

### PR DESCRIPTION
**Pull request by GeekJosh**
This PR adds support for a bootstrap4 theme. I have taken the approach of using FontAwesome for icons as this remains freely available.

### CODE REVIEW

**Comment by acerix** [comment]
Thanks for the PR

File: src/theme/Bootstrap4Theme.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Bootstrap4Theme.prototype.baseIconClass = 'fa'
+Bootstrap4Theme.prototype.iconClasses = {
+  close: 'fa-times',
+  prev: 'fa-chevron-left',
+  next: 'fa-chevron-right',
+  prevYear: 'fa-angle-double-left',
+  nextYear: 'fa-angle-double-right'
+}
+
+Bootstrap4Theme.prototype.iconOverrideOption = 'bootstrapGlyphicons'
```
[the remark this answers is not in the document]
Comment (GeekJosh) [reply src/theme/Bootstrap4Theme.ts]: That's great feedback, thank you. I'll update my PR later on today

[the remark this answers is not in the document]
Comment (joankaradimov) [reply src/theme/Bootstrap4Theme.ts]: Thanks!

**Comment by GeekJosh** [comment]
@joankaradimov 
- I've updated the list of themes, thanks for listing them for me
- The colour fix using `table-bordered` works well, this is what the outer container is using so the colour will always match
- I've added some extra styling around popovers so they should all e fixed now (Bootstrap overrides the `position` set by fullcalendar, so I've added a more specific CSS selector to retain `absolute`
- As suggested, icon overrides are now done using a `bootstrapFontAwesome` so that it doesn't conflict with the bootstrap3 theme

**Comment by GeekJosh** [comment]
The tests are failing on the repo due to an issue in locale-all - this is happening on the unmodified repo too. All tests for what I have added appear fine though (I have also added a manual test)

**Comment by arshaw** [comment]
really high quality PR and great feedback! i've merged this, made some tweaks, and will queue for release

**Comment by arshaw** [comment]
released in [v3.9.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v3.9.0) !

**Pull request merged**